### PR TITLE
Create readme.md with project overview

### PR DESCRIPTION
**Pull request by jparksecurity**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Hackathon Project: SenaiLabIntegration
+
+## Overview
+
+Welcome to the GitHub repository for our project developed during the 20-hour InovaTI HACKATHON. This project aims to integrate a laboratory testing machine into the lab's digital system.
```
Comment (lkasvr) [inline readme.md]: I think that we could describe what the laboratory does whit one word in the beginning even whether it was described below, like: cement laboratory or construction site laboratory

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## The Challenge
+
+During the Hackathon, participants were tasked with creating innovative solutions for real problems local companies have. Our team focused on solving the data integrity problem of a lab test result. Senai, a local lab, tests construction materials such as concrete to verify their quality. After testing the material, a lab technician has to read the result value on the machine's display and record the result in their system, which is error-prone.
+
+## Our Solution
+
+The **lab-integration** tracks a material testing machine's test result and automates a form submission for a new result on the lab's website. We wanted to write directly on the database, but the lab's IT department refused access. As we didn't know the database schema, we had to build a macro to fill the form on the lab's website, which the lab technician let us skim through.
+
+The **mock-testing-environment** mocks the lab's environment: running a test and recording the test result.
```
Comment (lkasvr) [inline readme.md:15]: Do you think that is necessary mention the lab-integration branch here too?

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+To get a local copy up and running, follow these simple steps:
+
+1. Clone the repository:
+
+```bash
+git clone https://github.com/lkasvr/senai-demo.git
+```
+
+### mock-testing-environment
```
Comment (lkasvr) [inline readme.md:39]: Could you merge the config instructions? Just to have the second step with both installation process and simplify and reduce the readme file. What do you think about it?

**Review by lkasvr — CHANGES_REQUESTED** [review]

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+## Overview
+
+Welcome to the GitHub repository for our project developed during the 20-hour InovaTI HACKATHON. This project aims to integrate a laboratory testing machine into the lab's digital system.
+
+## The Challenge
+
+During the Hackathon, participants were tasked with creating innovative solutions for real problems local companies have. Our team focused on solving the data integrity problem of a lab test result. Senai, a local lab, tests construction materials such as concrete to verify their quality. After testing the material, a lab technician has to read the result value on the machine's display and record the result in their system, which is error-prone.
+
+## Our Solution
```
Comment (lkasvr) [inline readme.md:11]: Could we tell about this approach as a goal for a demonstration? Then make it clear that our main solution to this challage works with another strategy (headless)

**Review by lkasvr — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (jparksecurity) [reply readme.md]: acknowledged

File: readme.md (hunk 2 above)
Comment (jparksecurity) [reply readme.md:15]: It's already there.

File: readme.md (hunk 3 above)
Comment (jparksecurity) [reply readme.md:39]: I think it's easier to read in the current way.

File: readme.md (hunk 4 above)
Comment (jparksecurity) [reply readme.md:11]: Add a note at the end of the readme.

File: readme.md (hunk 3 above)
Comment (lkasvr) [reply readme.md:39]: Ok, let's stay with the current way.

**Review by lkasvr — APPROVED** [review]

**Pull request merged**